scripts/load_configured_data

```diff
@@ -52,7 +59,15 @@ then
         usage
     else
         configPath=${1:-config/config.xml}
-        docker-compose up -d postgres django
+        if [ "${1:-}" = "--production" ]
+        then
+          export COMPOSE_FILE="docker-compose.yml:docker-compose.production.yml"
```

Comment:
It is unclear why variables are being exported here. Is the idea that they need to be available to `docker-compose` for one condition and not the other?